microsoftgraph/docs-conceptual/migration-steps.md

```diff
@@ -3,57 +3,89 @@ title: "Migrate from Azure AD PowerShell to the Microsoft Graph PowerShell SDK."
+- **Understand output objects**: Understand the change in output objects in Microsoft Graph PowerShell. The [Get-Member](/powershell/module/microsoft.powershell.utility/get-member) cmdlet will help you understand the objects.
+- **Optionally, understand the underlying API**
+  - Use [Find-MgGraphCommand](find-mg-graph-command.md) to find the API path a cmdlet calls.
+  - Use [Graph Explorer](https://developer.microsoft.com/en-us/graph/graph-explorer) to understand the underlying API calls.
+  - Reference [API reference content](/graph/api/overview).
 
 #### Example
 
-The equivalent of the Get-AzureADUser cmdlet is [Get-MgUser](/powershell/module/microsoft.graph.users/get-mguser?view=graph-powershell-1.0&preserve-view=true). This cmdlet has additional parameters that let you do more with it's output e.g the ConsistencyLevel parameter that allows you to do `Count`, `Filter`, `Search` among others.
+The equivalent of the `Get-AzureADUser` cmdlet is [Get-MgUser](/powershell/module/microsoft.graph.users/get-mguser?view=graph-powershell-1.0&preserve-view=true). This cmdlet has additional parameters that let you do more with its output. For example, the **-ConsistencyLevel** parameter that allows you to do `Count`, `Filter`, and `Search`.
+
+```powershell
+Connect-MgGraph -Scopes 'User.Read.All'
+Get-MgUser -ConsistencyLevel eventual -Count userCount -Filter "startsWith(DisplayName, 'a')" -Top 1
+```
 
 ### Limitations
 
-There are limitations that currently exist in Microsoft Graph PowerShell either by design or due to some functionality that is yet to be built in.
+There are limitations that currently exist in Microsoft Graph PowerShell, either by design or due to some functionality that is yet to be built in.
 
-- There is not yet an equivalent of -SearchString for Get-AzureADUser, Get-AzureADGroup commands. Use -Filter instead. For example, `Get-MgUser -Filter "DisplayName eq 'Lee Gu'"` returns the user whose display name is equal to the specified string.
-- Search does not yet work for any Azure AD commands.
-- You need to use hash tables to pass nested parameters. Here is a sample of [Nested parameters](https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/dev/samples/9-Applications.ps1#L28-L43).
-- **Pro-tip**: Use the Microsoft Graph PowerShell ConsistencyLevel parameter. It lets you do $count! This is best used for read-only scenarios and you need to be more careful when making changes. To learn more about the ConsistencyLevel parameter see [Advanced query parameters](/graph/aad-advanced-queries).
+- There isn't yet an equivalent of **-SearchString** for `Get-AzureADUser` and `Get-AzureADGroup` commands. Use **-Filter** instead. For example, `Get-MgUser -Filter "DisplayName eq 'Lee Gu'"` returns the user whose display name is equal to the specified string.
+- Search doesn't yet work for any Azure AD commands.
```

Comment:
This is not true -- Search works for many commands such as `Get-MgUser` -- as long as you use `ConsistencyLevel` of `Eventual`. The behavior is not the same as `SearchString` but arguably better.